datafusion/proto/src/logical_plan/mod.rs

```diff
@@ -353,12 +353,9 @@ impl AsLogicalPlan for LogicalPlanNode {
                             self
                         ))
                     })? {
-                        &FileFormatType::Parquet(protobuf::ParquetFormat {
```

Comment:
The plumbing for this override was actually incorrect, it would convert `false -> None`, the other overrides aren't present, and we plan to remove this override mechanism as part of #4349 so I just opted to remove it

Reply:
I agree serializing the same config options multiple times (once in the main session context and then once again as part of the file format) is undesirable for many reasons